Create issue templates (#655)
sindresorhus/got: .github/ISSUE_TEMPLATE/2-feature-request.md

---
name: "⭐ Feature request"
about: Suggest an idea for Got

---

> #### What's the idea?

Reviewer: The first question should be:

```
### What problem are you trying to solve?
```

Many times, people post ideas that sound like a bad idea, but the actual problem is valid, just needs a better solution. And sometimes the problem is already solved in a better way. That's why knowing the problem to solve is more important than the suggested idea.

...

> <!-- If it's possible to make a workaround, please include the code too! It'll help us when working on the feature. 🦄 -->

Reviewer: ```suggestion
<!-- Include a usage example of the feature. If the feature is currently possible with a workaround, include that too. -->
```

#### Checklist

> - [ ] I have read the documentation. <!-- (no such feature exists right now) -->

Reviewer: ```suggestion
- [ ] I have read the documentation and made sure this feature doesn't already exist.
```
